Dockerfile

```diff
@@ -1,7 +1,5 @@
 FROM alpine:3.5
 
-MAINTAINER ZappiStore DevOps <devops@zappistore.com>
```

Comment:
This should be replaced with:
`LABEL maintainer "Zappi DevOps <devops@zappistore.com>"`